examples/advanced/edge/README.md

```diff
@@ -82,9 +82,12 @@ You will then see the device start receiving the model from the server and compl
 The server will perform aggregation and proceed to the next round.
 After the configured rounds have finished, the training is complete!
 
-## [Optional] Local Proof-Of-Concept with DeviceSimulator: an End-to-end Cifar10 Example 
+## [Optional] Local Proof-Of-Concept with Simulated TaskProcessingDevice: an End-to-end Cifar10 Example 
```

Comment:
we can enhance this later, 
we at least have DeviceSimulator + Proxy mode and TPRunner widget mode.